components/brave_extension/extension/brave_extension/background/api/shieldsAPI.ts

```diff
@@ -163,6 +163,13 @@ export const setAllowScriptOriginsOnce = (origins: Array<string>, tabId: number)
     })
   })
 
+/**
+ * Open a prompt that allows the user to submit a report telling Brave that the current website is broken by Shields.
+ * @param {number} tabId ID of the tab whose contents are being reported
+ */
+export const reportBrokenSite = (tabId: number) =>
+  chrome.braveShields.reportBrokenSite(tabId)
```

Comment:
Just tried to follow the convention where other API calls were wrapped in this file - I think it might be strange to have just one API call be different from the rest.